training/training_loop.py

```diff
@@ -7,6 +7,7 @@
 """Main training script."""
 
 import numpy as np
+from dnnlib.comet_utils import CometLogger
```

Comment:
Add comment: 

# Don't move this line:
# comet_ml needs to be imported before ML frameworks, like tensorflow

Reply:
I common way that this type of integration can break is if someone changes the import order (eg, moving comet_ml import below tensorflow).